util/format.zig

```diff
@@ -2,23 +2,21 @@ const std = @import("std");
 const Color = @import("./color.zig").Color;
 const log = std.log.scoped(.zbench_format);
 
-pub fn duration(buffer: []u8, d: u64) ![]u8 {
+pub fn duration(buffer: []u8, nanoseconds: u64) ![]u8 {
```

Comment:
Alternatively, just using the stdlib's duration formatting is a good option too IMHO:
```zig
pub fn duration(buffer: []u8, ns: u64) ![]u8 {
    return try std.fmt.bufPrint(buffer, "{}", .{
        std.fmt.fmtDuration(ns),
    });
}
```

Reply:
Good point. Leveraging the std lib can reduce complexity and potential errors in this custom implementation.